src/models/logic/transaction.ts

```diff
@@ -536,6 +536,23 @@ export async function getTransactionsByAssetType(params: {
     }
 }
 
+export async function getTransactionsByBlockNumber(params: {
+    blockNumber: number;
+}) {
+    try {
+        return models.Transaction.findAll({
+            where: {
+                ...buildQueryForTransactions(params)
+            },
+            order: [["blockNumber", "DESC"], ["transactionIndex", "DESC"]],
+            include: fullIncludeArray
+        });
+    } catch (err) {
+        console.error(err);
+        throw Exception.DBError();
+    }
+}
```

Comment:
This function is too general purpose to get the count the # of txs for each tx type.
I mean, you can just use `count` function with something like `group: ["type"]`